app/toolbar/toolbar.component.ts

```diff
@@ -96,6 +76,13 @@ export class ToolbarComponent implements OnInit {
         (data: any) => this.currentUser = <User> data,
         (error: any) => this.errorMessage = <any> error.error
       );
+      
+      this.notificationService.getUnreadNotificationsCount()
+        .then(
+          (response: any) => this.numberOfUnreadNotifications = response
+        ).catch(
+          (error: any) => console.log(error)
+        );
```

Comment:
When marking a Notification as read, this counter does not update.

At least this counter should be updated on going to another site.